project.godot

```diff
@@ -41,6 +41,10 @@ move_down={
 ]
 }
 
+[layer_names]
+
+2d_physics/layer_2="Player"
```

Comment:
It looks like the player is actually on layer 1. Shouldn't layer 2 be for enemies? That's the layer being detected by the hurtbox.